### DISCUSSION TITLE
Init called twice without proper unload

### DISCUSSION THREAD

**Discussion by FelipeCMarani** [Friendly0Fire/GW2Radial#309] [Q&A]
I have this problem and I don't know how to solve it, could someone help me?
[gw2al_log.txt](https://github.com/Friendly0Fire/GW2Radial/files/8346486/gw2al_log.txt)

**Comment by Friendly0Fire** [comment]
Please actually detail your problem. What is happening aside from this log?

**Reply by FelipeCMarani** [reply]
good currently when I open the game it is not starting any addon, I downloaded version 1.4.3 of GW2-Addon-Manager and I reinstalled the game a few times thinking it was the problem of missing some file

**Reply by FelipeCMarani** [reply]
and I also made the registry change Registry::\HKEY_USERS\Software\Microsoft\Windows NT\CurrentVersion\AppCompatFlags\Layers NoteProperty E:\Guild Wars 2\Gw2-64.exe=$ IgnoreFreeLibrary<d3d9.dll>

**Reply by Friendly0Fire** [reply]
So first please don't make random registry edits, no addon will require that, nor Windows compatibility to be enabled.

Second, this looks like an addon manager/loader issue more than a Radial one. What other addons do you have installed?

**Reply by FelipeCMarani** [reply]
@Friendly0Fire I'm sorry for all this trouble, I put the game to run as admin and as win 7 compatibility and it solved it, thanks for the help <3
